build.rs

```diff
@@ -104,13 +141,23 @@ use serde_json::Value;
     )
     .unwrap();
 
-    for entry in fs::read_dir(test_cases).unwrap() {
-        let path = entry.unwrap().path();
+    let mut paths: Vec<_> = fs::read_dir(test_cases)
+        .unwrap()
+        .map(|e| e.unwrap().path())
+        .collect();
+    paths.sort();
+    for path in paths {
```

Comment:
@acmiyaguchi `fs::read_dir` doesn't have stable ordering, so I added this to fix the CI build.

Reply:
How did the ordering affect the output of `git status`? I imagine that this section could be done in parallel for each path item in a deterministic way.

Reply:
it effected the order of tests in the rust test files, not the order of `git status`. changing to `git diff --exit-code` was a matter of being able to see why the check failed instead of just if it failed.

Reply:
I understand -- great!